File: src/main/java/org/jdc/template/domain/other/individuallist/IndividualListManager.java
/*
 * IndividualListManager.java
 *
 * Generated on: 03/10/2014 11:15:44
 *
 */



package org.jdc.template.domain.other.individuallist;


import org.jdc.template.domain.DatabaseManager;

import javax.inject.Inject;

@javax.inject.Singleton
public class IndividualListManager extends IndividualListBaseManager {

    @Inject
    public IndividualListManager(DatabaseManager databaseManager) {
        super(databaseManager);
    }


}

File: src/main/kotlin/org/jdc/template/domain/other/individuallist/IndividualListManager.kt
/*
 * IndividualListManager.kt
 *
 * Generated on: 02/14/2016 07:02:10
 *
 */



package org.jdc.template.domain.other.individuallist

import org.jdc.template.domain.DatabaseManager


@javax.inject.Singleton
class IndividualListManager : IndividualListBaseManager {


    @javax.inject.Inject
    constructor(databaseManager: DatabaseManager): super(databaseManager) {
    }


}